bb.sh

```diff
@@ -11,6 +11,9 @@ AWS_ACCESS_KEY=$3
 AWS_SECRET=$4
 awsBucket=$5
 
+#bitbucket workspace (name or uuid in {uuid})
+bbworkspace=$6
```

Comment:
We got now six arguments. Maybe we should create config file instead of passing these arguments inline?